app/controllers/staff/events_controller.rb

```diff
@@ -4,13 +4,13 @@ def edit
   end
 
   def show
-    event_teammates = @event.event_teammates.includes(:user).recent
+    teammates = @event.teammates.includes(:user).recent
```

Comment:
Now that the staff listing is happening elsewhere, revaluate how the event#show is using teammates and rating_counts.  This might be removed entirely or can be cut back significantly.  

Also, if you do still need teammates, what sort of teammates and how do you need to change access to get the right ones?